src/lib/components/Actions/index.js

```diff
@@ -20,82 +21,17 @@ class Actions extends BaseElement {
       actions: {type: String},
       // Pipe-seperated handles of authors of this page, including "@" if e.g. a Twitter user
       authors: {type: String},
-      // Whether the Web Share API is supported
-      webShareSupported: {type: Boolean},
     };
   }
 
-  constructor() {
-    super();
-    this.webShareSupported = isWebShareSupported();
-  }
-
-  onWebShare() {
-    navigator.share({
-      url: this.shareUrl,
-      text: this.shareText,
-    });
-  }
-
-  onTwitterShare(e) {
-    e.preventDefault();
-    window.open(e.target.href, 'share-twitter', 'width=550,height=235');
-  }
-
   get shareUrl() {
     return window.location.href;
   }
 
-  get shareText() {
-    let authorText = '';
-
-    const authors = this._splitPipes(this.authors);
-    if (authors.length) {
-      // ListFormat isn't widely supported; feature-detect it first
-      if ('ListFormat' in Intl) {
-        const il = new Intl.ListFormat('en');
-        authorText = ` by ${il.format(authors)}`;
-      } else {
-        authorText = ` by ${authors.join(', ')}`;
-      }
-    }
-
-    return document.title + authorText;
-  }
-
   get shareTemplate() {
-    if (this.webShareSupported) {
-      return html`
-        <button
-          class="w-actions__fab w-actions__fab--share gc-analytics-event"
-          data-category="web.dev"
-          data-label="share, web"
-          data-action="click"
-          @click=${this.onWebShare}
-        >
-          <span>Share</span>
-        </button>
-      `;
-    }
-
-    // Otherwise, fall back to a Twitter popup.
-    const url = new URL('https://twitter.com/share');
-    url.searchParams.set('url', this.shareUrl);
-    url.searchParams.set('text', this.shareText);
-    return html`
-      <a
-        class="w-actions__fab w-actions__fab--share gc-analytics-event"
-        data-category="web.dev"
-        data-label="share, twitter"
-        data-action="click"
-        href="${url}"
-        target="_blank"
-        rel="noreferrer"
-        @click=${this.onTwitterShare}
-      >
-        <span>Share</span>
-      </a>
-    `;
+    const element = new Share();
```

Comment:
This is a little odd (instantiating an element class just to render it) but stranger things have happened.

As long as the two modes work (Twitter, a link, _and_ Web Share, a button) then I'm fine with this!